maintainers/maintainer-list.nix

```diff
@@ -9988,4 +9988,14 @@
     github = "pulsation";
     githubId = 1838397;
   };
+  govanify = {
```

Comment:
This list should be sorted alphabetically, the addition to maintainers should also happen in a seperate commit.

Reply:
Done!